_data/simple-icons.json

```
@@ -5472,6 +5472,11 @@
             "hex": "003A9B",
             "source": "https://indeed.design/resources"
         },
+        {
+            "title": "Inertia",
+            "hex": "FFFFFF",
```

```suggestion
            "hex": "9553E9",
```

@mchev We just got their approve to use `#9553e9` as primary color.